ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/behaviour/MultiMoveBehaviour.java

```diff
@@ -0,0 +1,16 @@
+package com.github.migi_1.Context.model.entity.behaviour;
+
+import java.util.ArrayList;
+
```

Comment:
I cannot explain that class, since it is Luka's. When his branch is merged with the master as well, there will be comments.